api/src/main/java/org/pmiops/workbench/monitoring/views/GaugeMetric.java

```diff
@@ -1,72 +1,60 @@
 package org.pmiops.workbench.monitoring.views;
 
-import com.google.common.collect.ImmutableSet;
+import com.google.common.collect.ImmutableList;
 import io.opencensus.stats.Aggregation;
+import io.opencensus.stats.Aggregation.LastValue;
 import io.opencensus.stats.Measure.MeasureLong;
-import io.opencensus.tags.TagKey;
 import java.util.Collections;
 import java.util.List;
-import java.util.Set;
-import org.pmiops.workbench.monitoring.attachments.Attachment;
+import org.pmiops.workbench.monitoring.attachments.MetricLabel;
 
 public enum GaugeMetric implements Metric {
   BILLING_BUFFER_PROJECT_COUNT(
       "billing_buffer_project_count",
       "Number of projects in the billing buffer for each status",
-      ImmutableSet.of(Attachment.BUFFER_ENTRY_STATUS)),
-  COHORT_COUNT("cohort_count", "Count of all cohorts in existence"),
-  COHORT_REVIEW_COUNT("cohort_review_count", "Total number of cohort reviews in existence"),
+      ImmutableList.of(MetricLabel.BUFFER_ENTRY_STATUS)),
+  COHORT_COUNT("cohort_count_2", "Count of all cohorts in existence"),
+  COHORT_REVIEW_COUNT("cohort_review_count_2", "Total number of cohort reviews in existence"),
   DATASET_COUNT(
-      "dataset_count",
+      "dataset_count_2",
       "Count of all datasets in existence",
-      Collections.singleton(Attachment.DATASET_INVALID)),
+      ImmutableList.of(MetricLabel.DATASET_INVALID)),
   USER_COUNT(
-      "user_count",
+      "user_count_2",
       "total number of users",
-      ImmutableSet.of(
-          Attachment.USER_BYPASSED_BETA, Attachment.USER_DISABLED, Attachment.DATA_ACCESS_LEVEL)),
+      ImmutableList.of(
+          MetricLabel.USER_BYPASSED_BETA,
+          MetricLabel.USER_DISABLED,
+          MetricLabel.DATA_ACCESS_LEVEL)),
   WORKSPACE_COUNT(
-      "workspace_count",
+      "workspace_count_3",
```

Comment:
Everything else was incremented to _2, but this was incremented to _3. Who should know about why this metric is different from all the others?

Reply:
These are all going to get scrapped at least one more time, so I didn't invest in consistency or name conventions yet. I need to circulate a convention where we have a way to give versions to the metric descriptors.

It appears that registering (in OpenCensus) a view corresponding to an existing metric is a no-op if that metric is actually live, but the Google api fails with a "metric exists" error if the metric has been deleted. Still sorting this out a bit.